intel_owl/tasks.py

```diff
@@ -581,6 +582,15 @@ def signatureinfo_run(
     ).start()
 
 
+@shared_task(soft_time_limit=40)
```

Comment:
I see that you've kept a lower time limit and added a catch block for `SoftTimeLimitExceeded`. I haven't used Speakeasy myself, but I think in certain cases the analysis might take a few minutes. So maybe we should change this to around 120.

Reply:
I don't think that speakeasy, with the number of api that have right now, will ever need more than 40 second. But yeah, increasing has no cons, so I will update it.